packages/fluentui/react-bindings/src/hooks/useIFrameFocusDispatch.ts

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+
+export const FUI_FRAME_EVENT = 'fuiframefocus';
+
+/**
+ * Watch the `document.activeElement` and dispatch a custom event if the current active element is a iframe.
+ */
+export const useIFrameFocusDispatch = (
+  enableFrameFocusDispatch: boolean,
+  targetDocument: Document,
+  pollDuration: number = 1000,
+) => {
+  const timeoutRef = React.useRef<ReturnType<typeof setTimeout>>();
+
+  React.useEffect(() => {
+    if (enableFrameFocusDispatch) {
+      timeoutRef.current = setInterval(() => {
+        const activeElement = targetDocument?.activeElement;
+        if (activeElement?.tagName === 'IFRAME') {
+          const event = new CustomEvent(FUI_FRAME_EVENT, { bubbles: true });
+          activeElement.dispatchEvent(event);
+        }
+      }, pollDuration);
+    }
+    return () => {
+      clearTimeout(timeoutRef.current as ReturnType<typeof setTimeout>);
```

Comment:
if targetDocument is provided, use it for clearing interval (something like targetDocument.clearTimeout)

Reply:
document cannot call clearInterval it is a window global method. If you really wanted to do smth like that it would have to be:
```js
document.defaultView.clearInterval
```